frontend/src/utils/serializeFilterParams.js

```diff
@@ -44,10 +43,13 @@ export default function serializeFilterParams(filters, descriptions) {
             key += "__recursive"
           else if (options.exactMatch)
             key += "__startswith"
+          else if (options.many_to_many)
```

Comment:
It is not really clear to me what is that many_to_many option. Does this represent a real option (we can actually set it to false ?) or is it simply a descriptor for the filter. Here both are used but i doubt from what i have seen that you actually have a choice (could possibly only use descriptor.many_to_many). I am also uncertain what it differentiate between. Is seem to be a way to say exact_match when there is optional filtering but nothing when there is no options ... Lastly it seem very specific to the project and sample case ... can't we use the preexisting options and descriptors to cover that case ? or maybe rename it so it's clear what it means ?

Reply:
Same. I dont understand the 'many_to_many' as an option AND as a description 
(it can be both? )
Ref: 
![image](https://user-images.githubusercontent.com/1617741/133159391-1d8d0c5d-bf07-4cb0-8cbc-21702c4bc9d6.png)